public/app/features/explore/utils/decorators.ts

```diff
@@ -1,214 +1,187 @@
+          getDisplayProcessor({
+            field,
+            theme: config.theme,
+            timeZone: data.request?.timezone ?? 'browser',
+          });
       }
 
-      data.tableFrames.sort((frameA: DataFrame, frameB: DataFrame) => {
-        const frameARefId = frameA.refId!;
-        const frameBRefId = frameB.refId!;
-
-        if (frameARefId > frameBRefId) {
-          return 1;
-        }
-        if (frameARefId < frameBRefId) {
-          return -1;
-        }
-        return 0;
-      });
-
-      const hasOnlyTimeseries = data.tableFrames.every(df => isTimeSeries(df));
-
-      // If we have only timeseries we do join on default time column which makes more sense. If we are showing
-      // non timeseries or some mix of data we are not trying to join on anything and just try to merge them in
-      // single table, which may not make sense in most cases, but it's up to the user to query something sensible.
-      const transformer = hasOnlyTimeseries
-        ? of(data.tableFrames).pipe(standardTransformers.seriesToColumnsTransformer.operator({}))
-        : of(data.tableFrames).pipe(standardTransformers.mergeTransformer.operator({}));
-
-      return transformer.pipe(
-        map(frames => {
-          const frame = frames[0];
-
-          // set display processor
-          for (const field of frame.fields) {
-            field.display =
-              field.display ??
-              getDisplayProcessor({
-                field,
-                theme: config.theme,
-                timeZone: data.request?.timezone ?? 'browser',
-              });
-          }
-
-          return { ...data, tableResult: frame };
-        })
-      );
+      return { ...data, tableResult: frame };
     })
   );
+};
 
 export const decorateWithLogsResult = (
-  state: ExploreItemState
-): MonoTypeOperatorFunction<ExplorePanelData> => inputStream =>
-  inputStream.pipe(
-    map(data => {
-      if (data.error) {
-        return { ...data, logsResult: null };
-      }
-
-      const { absoluteRange, refreshInterval } = state;
-      if (data.logsFrames.length === 0) {
-        return { ...data, logsResult: null };
-      }
-
-      const timeZone = data.request?.timezone ?? 'browser';
-      const intervalMs = data.request?.intervalMs;
-      const newResults = dataFrameToLogsModel(data.logsFrames, intervalMs, timeZone, absoluteRange);
-      const sortOrder = refreshIntervalToSortOrder(refreshInterval);
-      const sortedNewResults = sortLogsResult(newResults, sortOrder);
-      const rows = sortedNewResults.rows;
-      const series = sortedNewResults.series;
-      const logsResult = { ...sortedNewResults, rows, series };
-
-      return { ...data, logsResult };
-    })
-  );
-
-function isTimeSeries(frame: DataFrame, datasource?: string): boolean {
-  // TEMP: Temporary hack. Remove when logs/metrics unification is done
-  if (datasource && datasource === 'cloudwatch') {
-    return isTimeSeriesCloudWatch(frame);
```

Comment:
We should not be needing this as right now the new test is handling wide dataFrames. @kaydelaney just for sure, the cloudWatch check would also pass dataFrames with time/numeric/string columns as timeSeries dataFrames. Is that how it should work? How do we handle the string columns in that case?